### PR TITLE
refactor(progressbar): FRMW-270 progressbar LESS Vars

### PR DESCRIPTION
**Pull request by ssalinas24**
JIRA: https://jira.rax.io/browse/FRMW-270

- [x] Dev LGTM
- [x] Dev LGTM

### CODE REVIEW

File: src/styles/vars.less [hunk 1]
```diff
@@ -301,3 +301,22 @@
...
+ @progress-bg: #f5f5f5;
+ //** Default progress bar color
+ @progress-bar-bg: @light-blue;
+ //** Secondary progress bar color
+ @progress-bar-secondary-bg: @infoBlue;
+ //** Tertiary progress bar color
+ @progress-bar-tertiary-bg: @actionGreen;
+ @font-size-base: @appFontSize;
+ @font-size-large:         ceil((@font-size-base * 1.25));
+ @font-size-small:         ceil((@font-size-base * 0.85));
```
Comment (CITguy) [inline src/styles/vars.less]: Please reduce the above two lines to a single space between variable and value.

File: src/styles/vars.less [hunk 2]
```diff
@@ -301,3 +301,22 @@
...
+ //** Unit-less `line-height` for use in components like buttons.
+ @line-height-base:        1.428571429; // 20/14
```
Comment (CITguy) [inline src/styles/vars.less]: Please reduce to a single space between variable and value.

File: src/styles/vars.less [hunk 3]
```diff
@@ -301,3 +301,22 @@
...
+ //** Computed "line-height" (`font-size` * `line-height`) for use with `margin`, `padding`, etc.
+ @line-height-computed:    floor((@font-size-base * @line-height-base)); // ~20px
```
Comment (CITguy) [inline src/styles/vars.less]: Same as above.

File: src/styles/vars.less [hunk 4]
```diff
@@ -301,3 +301,22 @@
...
+@progress-bar-secondary-bg: @infoBlue;
+//** Tertiary progress bar color
+@progress-bar-tertiary-bg: @actionGreen;
+@font-size-base: @appFontSize;
+@font-size-large: ceil((@font-size-base * 1.25));
+@font-size-small: ceil((@font-size-base * 0.85));
+//** Unit-less `line-height` for use in components like buttons.
+@line-height-base: 1.428571429; // 20/14
+//** Computed "line-height" (`font-size` * `line-height`) for use with `margin`, `padding`, etc.
+@line-height-computed: floor((@font-size-base * @line-height-base)); // ~20px
```
Comment (ssalinas24) [inline src/styles/vars.less]: @CITguy Thank you for the feedback. I have corrected the spacing as you requested. As well as additional spacing that I saw on line: 228 - 231. This is ready for review again.

**Comment by CITguy** [comment]
LGTM

**Comment by bry4n** [comment]
LGTM

**Pull request merged**